docs/en/userguide/index.md

```diff
@@ -16,17 +16,17 @@ Make sure that your SilverStripe installation has the [versionfeed](http://addon
 
 There are two feeds that are automatically created for each page:
 
-* Page changes: This feed will display all published versions of the page, highlighting any additions or deletions with underscores or strikethroughs. It is accessible with the `changes` action - so `http://mysite.com/mypage/changes`
+* Page changes: This feed will display all published versions of the page. It is accessible with the `changes` action - so `http://mysite.com/mypage/changes`
```

Comment:
is the dropping of the highlighting and deletions a regression? I feel like that's still a thing.

Reply:
It is indeed, I looked at the XML instead of using a feed reader. Duh!